### PR TITLE
DOC: document NPY_DISABLE_CPU_FEATURES

### PR DESCRIPTION
**Pull request by mattip**
xref #22904

### CODE REVIEW

**Comment by mattip** [comment]
oy, I always forget to add `[ci skip]`  to documentation PRs.

File: doc/source/reference/simd/how-it-works.rst [hunk 1]
```diff
@@ -347,3 +347,21 @@ through ``--cpu-dispatch``, but it can also represent other options such as:
...
+
+Importing NumPy triggers a scan of the available CPU features from the set
+of dispatchable features. This can be further restricted by setting the
+environment variable ``NPY_DISABLE_CPU_FEATURES`` to a comma-, tab-, or
+space-separated list of features to disable. This will raise an error if
+parsing fails or if the feature was not enabled. For instance, on ``x86_64``
+this will disable ``AVX2`` and ``FMA3``::
+
+    NPY_DISABLE_CPU_FEATURES="AVX2,FMA3"
+
```
Comment (seiko2plus) [inline doc/source/reference/simd/how-it-works.rst:366]: Better to move it to https://github.com/numpy/numpy/blob/main/doc/source/reference/simd/build-options.rst#runtime-trace. This document will be removed soon or later after moving to meson.

Comment (mattip) [reply doc/source/reference/simd/how-it-works.rst]: done

**Comment by seiko2plus** [comment]
Thank you Matti, I follow it with pr to clarify more about the behavior of disabling implicit features.

**Pull request merged**